tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "types": ["reflect-metadata", "jest"],
+    "types": ["jest"],
```

Comment:
This was making `tsconfig.json` have a compilation error when opened on VSCode.

Reply:
Yeah I noticed this but I left it since I thought it was a local issue